lib/adapters/serializers/csv.js

```diff
@@ -0,0 +1,40 @@
+var _ = require('underscore');
+var base = require('./base');
+var csv = require('csv-write-stream');
+
+module.exports = base.extend({
+
+    initialize: function() {
+        this.header;
+    },
+
+    write: function(points) {
+        var self = this;
+
+        _.each(points, function(point) {
```

Comment:
That strategy is all over the UI, but you rarely see it on the backend for whatever reason.